.github/workflows/build.yml

```diff
@@ -14,24 +14,15 @@ jobs:
             vcpkg_triplet: x64-windows
             vcpkg_host_triplet: x64-windows           
             vcpkg_overlay_ports: overlay/windows;overlay/ports
-            vcpkg_packages_extras: angle libid3tag libmad qt5-winextras
+            vcpkg_packages_extras: libid3tag libmad
```

Comment:
The zip file seems to contain the Angle files libGLESv2.dll, libGLESv2.pdb. Can it be that some cache wasn't cleaned properly?

Reply:
I have now deleted the cache and started over.